Makefile

```diff
@@ -222,4 +231,39 @@ endif
 
 # Test all
 .PHONY: test-all
-test-all: test-account-creation test-ccs test-reuse test-awsfederatedaccountaccess
\ No newline at end of file
+test-all: test-account-creation test-ccs test-reuse test-awsfederatedaccountaccess
+
+# Test apply ou map cr
+.PHONY: create-ou-map
+create-ou-map:
+	# Create OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap_ou_mapping.tmpl | oc apply -f -
+
+# Test delete ou map cr
+.PHONY: delete-ou-map
+delete-ou-map:
+# Delete OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap_ou_mapping.tmpl | oc delete -f -
+
+# Test claim-account
+.PHONE: claim-account
+claim-account:
+	# Create account claim CR
+	@oc process -p NAME=${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD}-claim -p NAMESPACE=${NAMESPACE} -f hack/templates/aws_v1alpha1_accountclaim_cr.tmpl | oc apply -f -
+	# Wait for account to become claimed
+	@while true; do CLAIMED=$$(oc get account ${OSD_STAGING_1_ACCOUNT_CR_NAME_OSD} -n aws-account-operator -o json | jq -r '.status.claimed'); if [ "$$CLAIMED" == "true" ]; then break; fi; sleep 1; done
+
+# Test delete account-claim
+.PHONY: delete-account-claim
```

Comment:
Do we need this in addition to `delete-accountclaim`?

Reply:
same namespace issues